metaseq_cli/interactive_cli.py

```diff
@@ -102,12 +101,38 @@ def cli_main():
     # dumb defaults overriding
     parser.set_defaults(lr_scheduler=None, criterion=None)
     flat_launch_args = []
-    for s in LAUNCH_ARGS:
-        flat_launch_args += s.split()
+
+    if len(sys.argv) > 1:
+        try:
+            from metaseq_internal.constants import TRAINED_MODEL_CONFIG
+        except ImportError:
+            TRAINED_MODEL_CONFIG = {}
+        model_name = sys.argv[1]
+        env_name = "azure" if len(sys.argv) < 3 else sys.argv[2]
```

Comment:
should we give these things a proper argparse? I don't understand how the sys.argv works or how I'm supposed to use the API now.